docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -43,8 +43,8 @@ In the meantime, if any platform or service in Azure still doesn't support .NET
 
 ### Additional resources
 
-- **.NET Core Guide** \
-  [https://docs.microsoft.com/dotnet/core/index](../../../core/index.yml)
+- **.NET fundamentals** \
+  [https://docs.microsoft.com/dotnet/core/index](../../../fundamentals/index.yml)
```

Comment:
Link text needs to be changed too.
I also can't find index.yml in fundamentals folder. However, the build isn't reporting a warning for it. Am I missing something?